flaskapp.py

```diff
@@ -230,7 +234,11 @@ def to_dict(self, fields=[]):
             'zone_format': self.zone_format,
             'grid_primary': self.grid_primary,
             'grid_secondaries': self.grid_secondaries,
-            'ns_group': self.ns_group
+            'ns_group': self.ns_group,
+            'delegate_to': self.delegate_to,
+            'forwarding_servers': self.forwarding_servers,
+            'external_servers': self.external_servers,
+            'stub_members': self.stub_members
```

Comment:
Leave a trailing comma for cleaner diffs in the future:

```suggestion
            'stub_members': self.stub_members,
```